spec/models/review_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+# RSpec.describe Review, type: :model do
```

Comment:
this file should be removed, or more ideally, you should have some unit tests for the review model